src/char/char_clif.c

```diff
@@ -885,61 +888,143 @@ int chclif_parse_charselect(int fd, struct char_session_data* sd,uint32 ipl){
 	return 1;
 }
 
+enum e_charclif_error {							//tested with 0x6e in 20150914
+	ERROR_SAMENAME_EXIST = 0x0,					//Character name already exist
```

Comment:
That enum for clif are extracted from aegis. I prefer kinda keep their name there, just to have a kinda sync.